docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx

```diff
@@ -0,0 +1,619 @@
+
+## `fetch` API
+
+Next.js extends the native [`fetch` Web API](https://developer.mozilla.org/docs/Web/API/Fetch_API) `fetch` to allow you to allow you to configure the [caching](#caching-data) and [revalidating](#revalidating-data) behavior for each fetch request on the server. You can use `fetch` in [Server Components](/docs/app/building-your-application/rendering/server-components), [Route Handlers](/docs/app/building-your-application/routing/route-handlers), and [Server Actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations). For example:
+
+```tsx filename="app/page.tsx" switcher
+export default async function Page() {
+  const data = await fetch('https://api.example.com/...').then((res) =>
+    res.json()
+  )
+
+  return <></>
+}
+```
+
+```jsx filename="app/page.js" switcher
+export default async function Page() {
+  const data = await fetch('https://api.example.com/...').then((res) =>
+    res.json()
+  )
+
+  return <></>
+}
+```
+
+By default, `fetch` requests fetch fresh data. Using it will cause a whole route to be [dynamically rendered](/docs/app/building-your-application/rendering/server-components#dynamic-rendering) and data will not be cached.
+
+You can cache `fetch` requests by setting the `cache` option to `force-cache`. This means data will be cached, and the component will be [statically rendered](/docs/app/building-your-application/rendering/server-components#static-rendering-default):
+
+```js
+fetch('https://...', { cache: 'force-cache' })
+```
+
+Alternatively, if using [PPR](/docs/app/building-your-application/rendering/partial-prerendering), we recommend wrapping components that use `fetch` requests in Suspense boundaries. This will ensure only the component that uses `fetch` is dynamically rendered and streamed in, and not the whole page:
+
+```tsx filename="@/app/ui/cart.tsx switcher
+import { Suspense } from 'react'
+
+
+export default async function Cart() {
+  const res = await fetch('https://api.example.com/...')
+
+  return <></>
+}
+
+export default function Navigation() {
+  return (
+    <>
+      <Suspense loading={<LoadingIcon />}>
+        <Cart />
+      </Suspense>
+    <>
+  )
+}
+```
+
+See the [caching and revalidating](/docs/app/building-your-application/data-fetching/caching-and-revalidating) docs for more information.
+
+> **Good to know:** In Next.js 14 and earlier, `fetch` requests were cached by default. See the [upgrade guide](/docs/app/building-your-application/upgrading/version-15) for more information.
+
+### Request Memoization
+
+If you need to fetch the same data in multiple components in a tree, you do not have to fetch data globally and pass props down. Instead, you can fetch data in the component that needs it without worrying about the performance implications of making multiple requests for the same data.
+
+This is possible because `fetch` requests with the same URL and options are automatically memoized during a React render pass.
```

Comment:
Putting my "very, very beginner hat on", is it fair to assume I know what "memoized" means? Maybe there's a dependent clause that could go hear for clarity?

> This is possible because `fetch` requests with the same URL and options are automatically memoized during a React render pass, so only one fetch will be executed.